src/Indatus/Dispatcher/Services/CommandService.php

```diff
@@ -208,6 +212,23 @@ public function getRunCommand(
             if (is_string($scheduledCommand->user())) {
                 array_unshift($commandPieces, 'sudo -u '.$scheduledCommand->user());
             }
+
+            if (Config::get('dispatcher::flock.enabled')) {
+                $lockFilesDir = Config::get('dispatcher::flock.lockFilesDirectory');
+                if (!$lockFilesDir) {
+                    $lockFilesDir = "/tmp";
+                }
+                $lockFilesDir = realpath($lockFilesDir);
+
+                $lockFilesPrefix = Config::get('dispatcher::flock.lockFilesPrefix');
+                if (empty($lockFilesPrefix)) {
+                    $lockFilesPrefix = "laravel";
+                }
+                $commandName = $scheduledCommand->getName();
```

Comment:
This is great!  Can you move the logic related to reading/parsing config to the https://github.com/Indatus/dispatcher/blob/1.4/src/Indatus/Dispatcher/ConfigResolver.php to help keep this class cleaner?